packages/examples/src/appPlayground/common.ts

```diff
@@ -0,0 +1,32 @@
+/* --------------------------------------------------------------------------------------------
+* Copyright (c) 2024 TypeFox and others.
+* Licensed under the MIT License. See LICENSE in the package root for license information.
+* ------------------------------------------------------------------------------------------ */
+
+import * as vscode from 'vscode';
+import { getService, IWorkbenchLayoutService } from 'vscode/services';
+import { RegisterLocalProcessExtensionResult } from 'vscode/extensions';
+import { MonacoEditorLanguageClientWrapper } from 'monaco-editor-wrapper';
+import { ConfigResult } from './config.js';
+
+export const configurePostStart = async (wrapper: MonacoEditorLanguageClientWrapper, configResult: ConfigResult) => {
+    const result = wrapper.getExtensionRegisterResult('mlc-app-playground') as RegisterLocalProcessExtensionResult;
+    result.setAsDefaultApi();
+
+    // currently unused
+    await getService(IWorkbenchLayoutService);
```

Comment:
what is it for?

Reply:
unused, but yes, better remove and bring back when really needed.

Reply:
Done ⬇️